chalice/Makefile

```diff
@@ -2,6 +2,6 @@ include ../common.mk
 
 deploy:
 	git clean -df chalicelib
-	cp -R ../dss ../dss-api.yml ../gcp-credentials.json chalicelib
+	cp -R ../dss ../dss-api.yml "$(GOOGLE_APPLICATION_CREDENTIALS)" chalicelib
```

Comment:
The user may have set GOOGLE_APPLICATION_CREDENTIALS a filename other than gcp-credentials.json, as is expected by the chalice config. Consider explicitly copying GOOGLE_APPLICATION_CREDENTIALS to the intended name.